regression-test/data/load_p0/insert/test_insert_default_value.out

```diff
@@ -1,8 +1,15 @@
 -- This file is automatically generated. You should know what you did if you want to edit this
 -- !select1 --
-10	10000	10000000	92233720368547758	19223372036854775807	10.3	10.3
-10	10000	10000000	92233720368547758	19223372036854775807	10.3	10.3
+10	10000	10000000	92233720368547758	19223372036854775807	10.30	10.3
```

Comment:
why this pr will change `10.3` to `10.30`

Reply:
i think it is not this pr changed the result scale. the regression framework do not check precision and scale, so the test could pass even if scale has been changed